src/evergreen/task.py

```diff
@@ -31,6 +32,29 @@ def __init__(self, json, api):
         super(Artifact, self).__init__(json, api)
 
 
+class StatusScore(IntEnum):
```

Comment:
Could you add some docstrings to this class and the `get_task_status_score` method?